### PR TITLE
Fix/content reorg

### PR DESCRIPTION
**Pull request by Emanuel-Allely**
⚠ PLEASE DO NOT DELETE THE TEXT BELOW ⚠

## Pull request's description

*Add here a description of the modifications you made.*

## Checklist before merging (please do not edit)
You must review you work before to submit the review to others.

### Self-review of your content
Remember the content must be readable and understandable by someone else than yourself.
- [ ] From a technical point of view.
- [ ] From a grammatical point of view (spelling mistakes, typos, clarity, etc.), and following the guidelines at the bottom.

### External reviews of your content
- [ ] You requested a technical review.
- [ ] You requested a grammatical review.
- [ ] You validated with @K0rdan or @alexgeron-Luos that it is safe to merge.

### Some guidelines to keep in mind
- Colons (:), semi-colons (;), exclamation (!), and interrogation points (?) are not preceded by a space (like full stops and commas). E.g.: Colons!
- File names and/or address are put in *italic*. E.g.: The file _main.c_.
- Functions, variables, or more generally short codes are put between grave accents. E.g.: To obtain `code()`, type \`code()\`.
- Long codes are put into blocks of code with three grave accent on each side, and the language's name:<br />
\`\`\`c<br />
// Some C language <br />
\`\`\`<br />

- "Luos engine" has a upper case on the **L** for "Luos" and lower case on the **e** for "engine".
- We call it "Luos engine" as a proper name, and ***not*** "the Luos engine".
- Following that fashion, anything that's owned by "Luos engine" implies that we must use `'s` as the standard English rule to show the possessive case, e.g. "Luos engine's API".
- The names pipe, gate, inspector, sniffer, app or application, driver, etc. have a lower case and can be refereed with the determiner `the`. E.g.: The inspector.

### CODE REVIEW

**Comment by Emanuel-Allely** [comment]
I just change order and move/add/delete items on sidebars and navbars. Fix some CSS in the tutorial section too.

File: apps/documentation/docs/luos-technology/index.mdx [hunk 1]
```diff
@@ -7,24 +7,39 @@ custom_edit_url: null
 ## The most for the developer
 
 Luos provides a simple way to think your hardware products as a group of independant features. You can easily manage and share your hardware products' features with your team, external developers, or with the community.
-Luos engine is an open-source lightweight library that can be used on any MCU, leading to free and fast multi-electronic-boards products development. Choosing Luos to design a product will help you to develop, debug, validate, monitor, and manage it from the cloud.
+Luos engine is an <a href="https://github.com/Luos-io/luos_engine" target ="_blank" rel="external nofollow">open-source lightweight library</a> that can be used on any [MCU](../compatibility/mcu_demoboard), leading to free and fast multi-electronic-boards products development. Choosing Luos to design a product will help you to develop, debug, validate, monitor, and manage it from the cloud.
+
+<a href="/tutorials/get-started" class="pagination-nav__link" style={{ display: 'inline-block' }}>
```
Comment (K0rdan) [inline apps/documentation/docs/luos-technology/index.mdx:12]: Did you forget the `target="_blank"` here ? Else why don't use a Markdown link ?

File: apps/documentation/docs/luos-technology/index.mdx [hunk 2]
```diff
@@ -7,24 +7,39 @@ custom_edit_url: null
...
+Most of the embedded developments are made from scratch. By using Luos engine, you will be able to capitalize on the development you, your company, or the Luos community already did. The re-usability of features encapsulated in Luos engine [services](./services) will fasten the time your products reach the market and reassure the robustness and the universality of your applications.
+
+We created a community on <a href="https://discord.gg/luos" target ="_blank" rel="external nofollow">Discord</a> to create exchanges among developers from around the world. This allows anyone to share their projects, needs and ideas for developing edge and embedded projects. Everyone is welcome to join:
 
-Need dedicated help for your project? You can join the Luos community on <a href="https://discord.gg/luos" target ="_blank" rel="external nofollow">Discord</a> or <a href="https://www.reddit.com/r/Luos/" target ="_blank" rel="external nofollow">Reddit</a>.
+<a href="https://discord.gg/luos" target ="_blank" rel="external nofollow" class="pagination-nav__link" style={{ display: 'inline-block' }}>
```
Comment (K0rdan) [inline apps/documentation/docs/luos-technology/index.mdx:22]: `target ="_blank"` unnecessary whitespace

File: apps/documentation/docs/luos-technology/node/topology.mdx [hunk 3]
```diff
@@ -13,19 +13,19 @@ The nodes can access the position of every other node in the network at any time
...
 
 The [Routing Table](/docs/luos-technology/services/routing-table) is a feature of Luos allowing every <Tooltip def={customFields.node_def}>node</Tooltip> to own a "map" (or topology) of the entire network of your device. This map allows nodes to know their physical position and different functionalities, as well as to easily search and interact with the other nodes.
 
-The [Routing Table](/docs/luos-technology/services/routing-table) is designed and shared among all the nodes after a process that is called [detection](/docs/luos-technology/services/routing-table).
+The Routing Table is designed and shared among all the nodes after a process that is called [detection](/docs/luos-technology/services/routing-table).
 
-:::info
-More details about routing table on the [dedicated page](/docs/luos-technology/services/routing-table).
-:::
+<a href="/docs/luos-technology/services/routing-table" target ="_blank" class="pagination-nav__link" style={{ display: 'inline-block' }}>
```
Comment (K0rdan) [inline apps/documentation/docs/luos-technology/node/topology.mdx:22]: Replace all `target ="_blank"` by `target="_blank"`

**Review by K0rdan — CHANGES_REQUESTED** [review]
Replace all links from `[(.*)]\(\/docs\/(.*))` to [$1](/documentation/$2)

**Pull request merged**